### DISCUSSION TITLE
Add link rendered event

### DISCUSSION THREAD

**Discussion by arpanparpan** [Blazor-Diagrams/Blazor.Diagrams#388] [Q&A]
Hi. I want to start by saying what an awesome product this is. Truly amazing.

That said. 

I wonder if there is a way to see when all links in a Diagram has been rendered. 
I am showing somewhat 400 nodes with one link each created with Router = new OrthogonalRouter(), PathGenerator = new StraightPathGenerator(). However as these take some time to render I would like to prevent certain actions while rendering. 

If this can't be done today. Would you like me to implement it and create a pull request?

**Comment by zHaytam** [comment]
I'm sorry I did not see this discussion.

The only way I think this could be done is by waiting to receive ALL `OnAfterRender` events, but I'm not sure how accurate that would be
